src/main/java/hw_5/Task3/Point.java

```diff
@@ -0,0 +1,23 @@
+package hw_5.Task3;
+
+public class Point {
+
+    public int x;
```

Comment:
access modifiers are public?

Reply:
fixed

Reply:
not fixed yet
![Screenshot_3](https://github.com/user-attachments/assets/7e701db3-b556-4456-bd34-e1e57db329d7)

Reply:
fixed